backend/app/services/leaderboard.py

```diff
@@ -85,17 +85,12 @@ def from_str(precision):
 class Task:
     benchmark: str
     metric: str
-    normalized_metric: str
     col_name: str
 
 class Tasks(Enum):
-    # task_key in the json file, metric_key in the json file, name to display in the leaderboard
-    # task0 = Task("IFEVal-fr", "metric_name", "IFEVal-fr")
-    # task1 = Task("GPQA-fr", "metric_name", "GPQA-fr")
-    # task2 = Task("BAC-fr", "metric_name", "BAC-fr")
-    task0 = Task("community|gpqa-fr|0", "acc", "norm_acc", "GPQA-fr")  # On pourrait vouloir mettre "Connaissances"
-    task1 = Task("community|ifeval-fr|0", "norm_acc", "norm_acc", "IFEval-fr") # FIXME norm_acc should be acc # et "Suivi d'instructions"
-    task2 = Task("community|bac-fr|0", "bac-fr-qem", "bac-fr-qem", "bac-fr") # et "Suivi d'instructions"
+    task0 = Task("community|gpqa-fr|0", "acc", "GPQA-fr")  # On pourrait vouloir mettre "Connaissances"
+    task1 = Task("community|ifeval-fr|0", "prompt_level_strict_acc", "IFEval-fr") # FIXME norm_acc should be acc # et "Suivi d'instructions"
```

Comment:
Is it really `prompt_level_strict_acc` we want here or a mean of two accuracies?

Reply:
The mean is done afterwards.